project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -101,4 +101,46 @@ private static User createUser(String email, String nickname) {
 
     return new User(email, nickname, userID);
   }
+
+  private static Document createDocument(String name, String language, String hash, long userID) {
+      Entity docEntity = new Entity("Document");
+      ArrayList<Long> userIDs = new ArrayList<Long>();
+      
+      docEntity.setProperty("name", name);
+      docEntity.setProperty("language", language);
+      docEntity.setProperty("hash", hash);
```

Comment:
maybe rename to documentID, since we have a lot of hashes in our database

Reply:
resolved in latest commit